WTWSMS/common/minor_titles/WtWSMS_minor_titles.txt

```diff
@@ -132,20 +132,21 @@ title_sigecempa = {
 }
 
 title_court_celtic_poet = { # Celtic Poet/Bard
-	dignity = 0.25
+	dignity = 1
```

Comment:
One quarter of one dignity doesn't  show up in the game. Dignity represents how they are treated in terms of how titles are selected as primary.